tests/e2e/rancher-test.ts

```diff
@@ -1,6 +1,5 @@
 import { test as baseTest, Locator, expect as baseExpect } from '@playwright/test'
 
-import { isString } from 'lodash'
```

Comment:
You should be able to import without issue using: `import isString from 'lodash/isString;`

Reply:
I replaced it with `typeof`, no need for external library here.
Original problem was that I only added lodash/merge in package.json but imported lodash...

Reply:
Fair enough, I don't see any reason to keep it in there then.
What you have is essentially what `isString`, there are just a few more type checks but they may be unnecessary:
```js
/**
 * Checks if `value` is classified as a `String` primitive or object.
 *
 * @static
 * @since 0.1.0
 * @memberOf _
 * @category Lang
 * @param {*} value The value to check.
 * @returns {boolean} Returns `true` if `value` is a string, else `false`.
 * @example
 *
 * _.isString('abc');
 * // => true
 *
 * _.isString(1);
 * // => false
 */
function isString(value) {
  return typeof value == 'string' ||
    (!isArray(value) && isObjectLike(value) && baseGetTag(value) == stringTag);
}
```